### DISCUSSION TITLE
Help!!

### DISCUSSION THREAD

**Discussion by LLeaXX** [microsoft/vscode-cpptools#13008] [Q&A] [answered]
Hello everyone. I tried to write a code in c++ in vs code, but when i click on "run" it shows me some weird symbols and chinese characters. I have c/c++ extension installed by the way.
![2024-11-30_16-26-54](https://github.com/user-attachments/assets/cc1e1bfb-839b-4581-8680-caa2fcce25f4)

**Comment by AumGupta** [comment, marked as answer]
Hey @LLeaXX,
This is rare to encounter but I have seen such cases. The problem is that the cause of this problem is difficult to identify. 

I would suggest you run the `course.cpp` file via command line first to see if it runs.

In command prompt go to the directory of cource.cpp:
``` bash
cd "c:\Users\3yb\Desktop\c++\"
g++ course.cpp -o course
.\course.exe
```

If it still doesn't work, I will suggest you consider reinstalling the compiler.

**Reply by LLeaXX** [reply]
Thank you SO MUCH! Have a great day!!
